apps/src/music/views/MusicView.jsx

```diff
@@ -58,10 +65,15 @@ class UnconnectedMusicView extends React.Component {
   static propTypes = {
     appOptions: PropTypes.object,
     appConfig: PropTypes.object,
-    levels: PropTypes.array,
-    currentLevelIndex: PropTypes.number,
+    /**
+     * True if Music Lab is being presented from the Incubator page (i.e. under /projectbeats),
+     * false/undefined if as part of a script or standalone level.
+     * */
+    inIncubator: PropTypes.bool,
```

Comment:
As discussed, Music Lab doesn't really know it's in an incubator, so I wonder if there's an alternate name here.  Maybe `isStandaloneApp`?  We could also adjust this later as we rationalize all of our variant naming.

Reply:
Yeah, unfortunately the issue with 'standalone' as Molly pointed out is that we often use it to refer to "projects/<lab>" pages. We just need some indicator so MusicView can decide to show the saving spinner or not. But yep as discussed we can adjust this later, and we don't expect the incubator page to stick around for too long, so I'll go ahead and ship with this for now.